### PR TITLE
[plugin-mobile-app] Add step to clear field

### PR DESCRIPTION
**Pull request by uarlouski**

### CODE REVIEW

File: docs/modules/plugins/pages/plugin-mobile-app.adoc [hunk 1]
```diff
@@ -321,6 +321,36 @@ Scenario: Type text
 When I type `Bob` in field located `accessibilityId(username)`
 ----
 
+=== Clear text
```
Comment (valfirst) [inline docs/modules/plugins/pages/plugin-mobile-app.adoc:324]: Clear field ?

File: vividus-plugin-mobile-app/src/main/java/org/vividus/mobileapp/action/KeyboardActions.java [hunk 2]
```diff
@@ -75,4 +76,23 @@ public void typeText(WebElement element, String text)
...
+     * <ol>
+     * <li>check if the element's text is empty</li>
+     * <li>if the element's text is not empty then clear the element, otherwise do nothing</li>
+     * </ol>
+     * @param element element to clear, must not be {@code null}
+     */
+    public void clearText(WebElement element)
+    {
+        String elementText = element.getText();
+        if (!StringUtils.isEmpty(elementText))
```
Comment (valfirst) [inline vividus-plugin-mobile-app/src/main/java/org/vividus/mobileapp/action/KeyboardActions.java:93]: why is this check needed?

Comment (uarlouski) [reply vividus-plugin-mobile-app/src/main/java/org/vividus/mobileapp/action/KeyboardActions.java:93]: getText is ~5 times faster than clear command, so if field is empty we will not spend time on clearing it

**Review by valfirst — CHANGES_REQUESTED** [review]

File: vividus-plugin-mobile-app/src/main/java/org/vividus/mobileapp/action/KeyboardActions.java (hunk 2 above)
Comment (uarlouski) [reply vividus-plugin-mobile-app/src/main/java/org/vividus/mobileapp/action/KeyboardActions.java:79]: Android platform: ~300ms vs ~1600ms

**Review by ikalinin1 — APPROVED** [review]

File: vividus-plugin-mobile-app/src/main/java/org/vividus/mobileapp/action/KeyboardActions.java (hunk 2 above)
Comment (ikalinin1) [reply vividus-plugin-mobile-app/src/main/java/org/vividus/mobileapp/action/KeyboardActions.java]: So in the most common case it is 1600 + 300 = 1800 ? :)

**Review by valfirst — APPROVED** [review]

**Pull request merged**